config/generateRunFile.coffee

```diff
@@ -414,7 +414,7 @@ generateDev = (KONFIG, options) ->
     }
 
     function runCountlyDocker () {
-        docker run -d -p $KONFIG_COUNTLY_APIPORT:80 --name=countly countly/countly-server:16.06
+        docker run -d -p $KONFIG_COUNTLY_APIPORT:80 --env 'COUNTLY_PATH=/countly' --name=countly koding/countly-server:docker-configure
```

Comment:
this docker image will be replaced with original one once they merge & release https://github.com/Countly/countly-server/pull/291